File: fib.c
#include <stdio.h>

int fibonacci(int n){
    if (n==0) return 1;
    if (n==1) return 1;
    return fibonacci(n-1) + fibonacci(n-2);
}

int main(){
    int n = 10;
    int i;
    for (i=0; i<n; i++){
        printf("%d ",fibonacci(i));
    }
    return 0;
}

File: fib.cs
using System;
public class fibonacci {
    static int fib(int n){
        if (n==0) 
            return 0;
        if (n==1)
            return 1;
        return fib(n-1)+fib(n-2);
    }
    public static void Main(string[] args){
        System.Console.WriteLine(fib(30));
    }
}
